http/http-advanced/src/test/java/io/quarkus/ts/openshift/http/AbstractHttpTest.java

```diff
@@ -164,6 +163,29 @@ public void nonAppRedirections() {
         }
     }
 
+    @Test
+    @Disabled("blocked by: QUARKUS-781")
+    public void microprofileHttpClientRedirection(){
+        assertThat(200, equalTo(healthClient.health().getStatus()));
```

Comment:
This test will never work when running in OpenShift as `@RestClient` annotation is not injected.

Reply:
;) Now works!